src/components/TypeBase/TypeBase.module.scss

```diff
@@ -397,3 +413,111 @@ h6,
     }
   }
 }
+
+/* ------------------------ Start Type2 ------------------------ */
+.Caption2 {
+  &.Theinhardt {
+    $font-size: 14;
```

Comment:
@keldonia I think this might be connected with the recent Figma spec you showed me that was `13.5px`. Btw, let's push back on design when the Figma lands on a half pixel.

But, it seems this is aligned with the sizes we were seeing there. Just thought you should see this PR and this section since we IM'd about it.

Reply:
@roblevintennis yes, smaller for the most part but not across the board. Some viewport breakpoints where the font size changed have been removed. I am working off of: https://docs.google.com/spreadsheets/d/1LiKYSuW5Lfh24OjT8e0WgNLcxuAZrFlbzmw_NJjUNOo/edit?usp=sharing